webapp/graphql/User.js

```diff
@@ -153,3 +153,14 @@ export const checkSlugAvailableQuery = gql`
     }
   }
 `
+
+export const localeMutation = () => {
+  return gql`
+    mutation($id: ID!, $locale: String) {
+      UpdateUser(id: $id, locale: $locale) {
+        id
+        locale
+      }
+    }
+  `
+}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-24T06:44:39Z" title="Thursday, October 24th 2019, 8:44:39 am +02:00">Oct 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -153,3 +153,14 @@ export const checkSlugAvailableQuery = gql`
     }
   }
 `
+
+export const localeMutation = () => {
+  return gql`
+    mutation($id: ID!, $locale: String) {
+      UpdateUser(id: $id, locale: $locale) {
+        id
+        locale
+      }
+    }
+  `
+}
```


is there a reason we can't just use the `UpdateUser` mutation?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-24T06:49:27Z" title="Thursday, October 24th 2019, 8:49:27 am +02:00">Oct 24, 2019</time>_

---


i took the embed code as my template. i also wondered if a simple user update wouldn't work, but i just wasn't sure.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-24T06:50:41Z" title="Thursday, October 24th 2019, 8:50:41 am +02:00">Oct 24, 2019</time>_

---


sorry, I guess we don't have an `UpdateUser` mutation?? we have the `allowEmbedIframesMutation` in`graphql/User.js` as well, and then we have `UpdateUser` mutations scattered throughout the `webapp`....
I'd really like it if we could refactor those all into one `UpdateUser` mutation in `graphql/User.js`, but if you want to do that in a separate `PR`, I'm ok with that